hasher-matcher-actioner/terraform/webapp/main.tf

```diff
@@ -1,7 +1,7 @@
 # Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved
 
 resource "aws_s3_bucket" "webapp" {
-  bucket = "${var.prefix}-hma-webapp"
+  bucket = "${var.organization}-${var.prefix}-hma-webapp"
```

Comment:
Why are both organization and prefix needed here? Wouldn't prefix be unique already? If wanting to share between prefixes, would need to remove prefix?

Reply:
@Dcallies From the docs:

An Amazon S3 bucket name is globally unique, and the namespace is shared by all AWS accounts. This means that after a bucket is created, the name of that bucket cannot be used by another AWS account in any AWS Region until the bucket is deleted.

So, similar to the cognito domain name, organization is introduced so we can have fb-prod-hma-webapp and snap-prod-hma-webapp.